eclipse-collections/src/main/java/org/eclipse/collections/impl/multimap/list/SynchronizedPutFastListMultimap.java

```diff
@@ -170,4 +170,9 @@ public void value(K key, MutableList<V> list)
     {
         return this.collectValues(function, FastListMultimap.<K, V2>newMultimap());
     }
+
+    public MutableListMultimap<K, V> asSynchronized()
+    {
+        return SynchronizedMutableListMultimap.of(FastListMultimap.newMultimap(this));
```

Comment:
Guess this line was changed after @motlin 's comment? Actually if you look inside `SynchronizedMutableListMultimap.of()`, this is already converted to non-synchronized list multimap.

```java
return new SynchronizedMutableListMultimap<K, V>(Multimaps.mutable.list.withAll(multimap));
```

Not sure if it was already like this when @motlin commented or it got changed afterwards. Anyways I suppose this line doesn't need to to wrap with `FastListMultimap.newMultimap()` anymore.

Same cases for other SynchronizedPut* classes.